.github/workflows/trigger-webui.yml

```diff
@@ -0,0 +1,57 @@
+# This workflow checks if the PR affects Anaconda (changes to pyanaconda folder),
+# polls the packit COPR until it has the current PR version
+# available, and then test-triggers an "anaconda PR" scenario.
+#
+# This workflow is inspired from:
+# https://github.com/cockpit-project/cockpit/blob/main/.github/workflows/trigger-anaconda.yml
+
+name: Anaconda Web UI
+on:
+  pull_request_target:
+    # only API changes might affect the Web UI
+    paths:
+      - pyanaconda/**
```

Comment:
I think you should include `anaconda.py` and probably ignore `pyanaconda/ui/gui` and `pyanaconda/ui/tui`.

Btw. is this about making sure that these changes don't break the Web UI, or also about monitoring changes of the behaviour (that might break the tests, but not the Web UI)? Otherwise, it might be good to monitor the configuration files as well (`data/anaconda.conf`, `data/profile.d`, `data/conf.d`).

Reply:
I did not file a way to exclude files yet. :/ I will include anaconda.py

This is also about monitoring the Web UI tests. For example, let's say that you do some changes that the device descriptions from the API are now different. This does not break the Web UI itself, but the tests will need adjustment, and it's great that we will get notified for this early on the PR stage. 

I will add the conf files as well, good point.

Reply:
> I did not file a way to exclude files yet. :/ I will include anaconda.py

Oh, I think you can use the `paths-ignore` keyword (see https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-excluding-paths). I have checked that before to make sure I am not asking of unicorns.

Reply:
It tried this already, but seems paths and paths-ignore properties are mutually exclusive. So using both will not work. Maybe I should better then use  paths-ignore: and go with the exclude list.

Reply:
I have updated my reply, because there is a third option that uses the `!` prefix.